<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HBWalkingDialog</class>
 <widget class="QDialog" name="HBWalkingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>632</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="BT_ONE_LEG_READY">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Stance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>100</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Time(s)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_TIME">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0.8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WALK_TEST">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Position Walking</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_TEST_STOP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_STEP_STRIDE">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_ONE_LEG_READY_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Stance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_TEST">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_HOME_POS">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Home Pos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>100</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Stride(m)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_WALK_READY">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Walk Ready</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>No Of Step</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_NO_OF_STEP">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>91</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_DATA_SAVE">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>data save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_READY_TO_WALK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Ready to Walk</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
